ext/opentelemetry-instrumentation-starlette/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## Unreleased
+
+## 0.8b0
+
+Released 2020-05-27
+
+- Add ASGI middleware ([#716](https://github.com/open-telemetry/opentelemetry-python/pull/716))
```

Comment:
I'm guessing this was copied over from ASGI and needs to be changed to starlette